include/crocoddyl/multibody/contacts/contact-1d.hxx

```diff
@@ -9,17 +9,36 @@
 namespace crocoddyl {
 
 template <typename Scalar>
-ContactModel1DTpl<Scalar>::ContactModel1DTpl(boost::shared_ptr<StateMultibody> state, const pinocchio::FrameIndex id,
-                                             const Scalar xref, const std::size_t nu, const Vector2s& gains)
-    : Base(state, 1, nu), xref_(xref), gains_(gains) {
+ContactModel1DTpl<Scalar>::ContactModel1DTpl(boost::shared_ptr<StateMultibody> state, 
+                                             const pinocchio::FrameIndex id,
+                                             const Scalar xref, 
+                                             const std::size_t nu, 
+                                             const Vector2s& gains, 
+                                             const std::size_t& type, 
+                                             const pinocchio::ReferenceFrame pinRefFrame)
+    : Base(state, 1, nu), xref_(xref), gains_(gains), type_(type), pinReferenceFrame_(pinRefFrame) {
   id_ = id;
+  if(type_ < 0 || type_ > 2){
```

Comment:
This doesn't follow the code format. Please run the code formatter as described in the wiki

Reply:
addressed in [fd58261](https://github.com/loco-3d/crocoddyl/pull/1054/commits/fd582614a34b626d8cb0751df8132d31f643845b)